modules/doc/content/newsletter/2025/2025_01.md

```diff
@@ -14,16 +9,97 @@ convergence and divergence criteria. The parsed expression may include other
+This is intended for being able to restrict the solid mechanics equations to only part of the subdomain. Simulations using multiple solid
+mechanics `Physics` will automatically create the displacement variables using the union of all the block restrictions, if `add_variables` is
+set in the shared `[Physics/SolidMechanics/QuasiStatic]` block. Simulations with other `Physics` deriving or creating the `QuasiStaticSolidMechanicsPhysics`
+programmatically (such as in BISON), should likely create the displacement variables in a `Variables` block.
+
+The method used by the four-noded shell elements ([ShellElements.md]) to define local coordinate systems has been revised. In the revised, the user provides a reference direction for the first local coordinate, and that is then projected onto the plane of the shells and used as the first local coordinate direction. An AuxKernel has been added to visualize the local coordinate directions. These changes, together with the correction of a longstanding bug with the calculation of the normal direction, enable the use of these elements for accurate computation of the response of curved shells, as confirmed by multiple regression tests that agree well with published analytical solutions.
+
+The [PressureAction.md] is now controllable. It can be activates and de-actived using the [Controls system](syntax/Control/index.md).
+The +enable+ status is propagaged to the boundary conditions that the action creates.
+
+## XFEM
+
+A new [MeshCut2DFractureUserObject.md] was added, it is a 2D mesh based cutting object that will nucleate and grow cracks in XFEM using fracture integrals computed by the Solid Mechanics [DomainIntegralAction.md]. The fracture toughness determining when a crack will grow can be defined as a spatially varying field and sampled using the [CrackFrontNonlocalScalarMaterial.md] vectorpostprocessor that is used as an input in the MeshCut2DFractureUserObject.
+
 ## libMesh-level Changes
 
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
 
-- MooseDocs SQA has the ability to check against multiple test specification file names (for example, `tests` as well
-  as `assessments`) when performing requirement reporting and checks, using the `specs:` parameter
-  in `sqa_reports.yml`.
+- MooseDocs SQA has the ability to check against multiple test specification file names (for example,
+  `tests` as well as `assessments`) when performing requirement reporting and checks, using the
+  `specs:` parameter in `sqa_reports.yml`.
+- The [ElementMaterialSampler.md] can now be used to sample only selected material properties rather than all properties
+  from a single `Material`. This can also be used to sample from properties defined by separate `Materials`.
+- A check was added to prevent infinite recursion when using the same input file as part both the parent and
+  child application.
```

Comment:
```suggestion
- A check was added to prevent infinite recursion when using the same input file as part of both the parent and
  child applications.
```